src/protocol/sort/reshare.rs

```diff
@@ -100,9 +103,6 @@ impl<F: Field> Reshare<F> for MaliciousContext<'_, F> {
         record_id: RecordId,
         to_helper: Role,
     ) -> Result<Self::Share, Error> {
-        use crate::protocol::context::SpecialAccessToMaliciousContext;
-        use crate::secret_sharing::ThisCodeIsAuthorizedToDowngradeFromMalicious;
-
```

Comment:
I'm not sure you want to do that. @martinthomson - is there a specific reason why these use statements were formerly located here?

Reply:
Ah, I just wanted to ensure that these were narrowly scoped.  They really shouldn't be moved to where they went here though.

Reply:
oops i thought they were misplaced.. Let me bring it back here

Reply:
By mistake i git commit --amend and force pushed the changes :( but it should be in correct place now